js/scripts.js

```diff
@@ -459,4 +459,39 @@ function showPopupWindow() {
 
 showPopupWindow(); 
 
-} //end of for desktop version
\ No newline at end of file
+} //end of popup window for desktop version
+
+
+
+
+// Contact Form Validation
+
+const emailInput = document.getElementById('email-input');
+console.log(emailInput.value);
+let emailPattern =  /^[a-z0-9-]+@[a-z0-9]+\.[a-z0-9.]+$/;
+
+//console.log("node value= "+emailInputValue.value);
```

Comment:
Kindly remove commented pieces of code in order to make the code neater. You can always go back in commit history to find them.